packages/sitecore-jss-angular/package.json

```diff
@@ -11,7 +11,7 @@
     "generate-docs": "npx typedoc --plugin typedoc-plugin-markdown --readme none --out ../../ref-docs/sitecore-jss-angular src/public_api.ts --githubPages false"
   },
   "engines": {
-    "node": ">=18"
+    "node": ">=20"
```

Comment:
We can't change it to Node 20, since other packages continue to use Node 18

Reply:
make sense.